worli.sh

```diff
@@ -1,266 +1,259 @@
     exit 1
 fi
-
 if ! command -v parted &> /dev/null
 then
-    echo "- parted package not installed. install it (for debian and ubuntu its 'sudo apt install parted', for arch it's also parted)"
+    echo "- 'parted' package not installed. Install it (For Debian and Ubuntu, run 'sudo apt install parted'; For Arch, run 'sudo pacman -S parted')"
     exit 1
 fi
-
 if ! command -v mkfs.ntfs &> /dev/null
 then
-    echo "- mkfs.ntfs command not found. install ntfs support somehow (like ntfs-3g and ntfsprogs)"
+    echo "- 'mkfs.ntfs' command not found. Install NTFW support somehow (such as ntfs-3g and ntfsprogs)"
     exit 1
 fi
 
-echo -e "\e[0;31mDO THE PRE STUFF BEFORE CONTINUING\e[0m"
-
-echo " "
+echo -e "\e[0;31mDO THE PREREQUISITES STUFF BEFORE CONTINUING\e[0m"
 read -p "Press any key to continue..."
 echo " "
 
-echo "what's the path to the 'win.iso'?"
-read -r -p "[/*] eg '~/win.iso': " iso
-
-echo " "
-echo "do you want to use a custom wim?"
+echo "What's the path to the 'win.iso'?"
+read -r -p "[/*] E.g. '~/win.iso': " iso
 echo " "
 
+echo "Do you want to use a custom WIM?"
 read -r -p "[N/Y]: " input
+echo " "
 case $input in
+
     [yY][eE][sS]|[yY])
-    
+
     echo "what's the path to the 'install.wim'?"
-    read -r -p "[/*] eg '~/install.wim': " wim
-    
+    read -r -p "[/*] E.g. '~/install.wim': " wim
+
     echo " "
     if [ -f $wim ]; then
-        echo "install.wim found"
-    else 
-        echo "install.wim does not exist. abort."
+        echo "'install.wim' found"
+    else
+        echo "'install.wim' does not exist. Abort."
         exit 1
     fi
- ;;
+    ;;
+
     [nN][oO]|[nN])
+
     echo " "
-    echo "no custom wim then"
+    echo "No custom WIM then"
+    ;;
 
-       ;;
     *)
- echo "Invalid input..."
- exit 1
- ;;
+
+    echo "Invalid input"
+    exit 1
+    ;;
 esac
 
 echo " "
 if [ -f $iso ]; then
-    echo "win.iso found"
-else 
-    echo "win.iso does not exist. abort."
+    echo "'win.iso' found"
+else
+    echo "'win.iso' does not exist. Abort."
     exit 1
 fi
-echo " "
-
-echo  "what /dev/* is your drive? NOTE: all your data on the selected disk will be erased, proceed with caution"
-echo  "--------------------------------------------------------------------------------------------------------"
 
+echo " "
+echo "What /dev/* is your drive? NOTE: ALL your data on the selected disk will be ERASED, proceed with caution!"
+echo "---------------------------------------------------------------------------------------------------------"
 parted -l
+read -r -p "[/dev/*] E.g. 'sdb', 'mmcblk0': " disk
 
-read -r -p "[/dev/*] eg 'sdb', 'mmcblk0': " disk
-
-echo "you have selected '$disk' is this correct?"
+echo "You have selected '$disk', is this correct?"
 read -r -p "[Y/N]: " input
 case $input in
     [yY][eE][sS]|[yY])
- echo "ok $disk it is"
- ;;
+    echo "OK '$disk' it is then"
```

Comment:
idk caps ok is kinda boomer in my eyes dont ya think

Reply:
~~it is the only correct grammar~~ tho i agree, will change it back